### ISSUE TITLE
add more thermostat and feature - Window Detection

### ISSUE THREAD

**Issue by GPL71** [maccoylton/esp-homekit-wired-thermostat#4]
Hello thank you for your project!

2 questions:

1 is it possible add more than 1 thermostat? I need of 5 devices for a hydronic floor heating
2 Could you add a daily scheduling?

Thank you

**Comment by maccoylton** [comment]
Homekit doesn't natively support setting schedules on an accessory. Apps from companies like Onvis or Eve do support this but use custom characteristics that are undocumented and custom apps to control those. So without either reverse engineering those or writing my own custom app for iOS it's not easy. 

Personally I use Apple's standard automations to schedule my thermostat, this of course depends on a reliable WiFi and a home hub. 


In terms of multiple thermostats , what behavior or rules would you want to implement?

**Comment by GPL71** [comment]
Thank you for your reply

I have 5 rooms with hydronic underfloor heating. I need to install a thermostat in every room. Can I pair all to homekit?

I need to set multiple temperature thresholds during the day or night.

Lower temperature at night and higher in the morning or lunch and afternoon and evening

Another interesting power-saving feature would be a routine that recognizes an open window. In this case the thermostat turn to off and after 30 or 60 minutes turn on itself.

Have a good day

**Comment by maccoylton** [comment]
You can pair as many as you likes, different temperatures and times etc can be done from the Home App automation section. You cans set them all individually and/or group them and set the trigger for the group. 

Window detection should be ok to add, something like 15 minutes to off after a couple of degree temperature drop, I'll add it to my enhancement list.

**Comment by GPL71** [comment]
Great!!! Many thanks.
I'll wait your update to test all....

**Comment by borissov** [comment]
Is there a different name for each thermostat, as well as a different MDNS host? Does it support NTP time synchronization for the clock? Can you change thermostat mode (manual/time based) from HomeKit?

**Comment by maccoylton** [comment]
Yes it creates a different host and name for each thermostat, Yes it uses NTP to set the clock, sometime for a reason I can't fathom this doesn't; always work,  yes you can change the mode. TBH I use Homekit automation to execute scheduling  and it's been reliable

**Comment by borissov** [comment]
Thank you!
